### ISSUE TITLE
Error -1 when calling clGetDeviceIDs for number of devices.

### ISSUE THREAD

**Issue by AX-Turbo** [wolf9466/wolf-xmr-miner#15]
**"Error -1 when calling clGetDeviceIDs for number of devices."**
Using 3 graphic cards (1: R9 290x, 2: R9 Nano, 3: R9 Nano). Whats wrong with my config file?

{
	"Algorithms":
	[
		{
			"name": "CryptoNight",
			"devices":
			[
				{
					"index": 0,
					"corefreq": 1050,
					"memfreq": 1250,
					"fanspeed": 60,
					"powertune": 20,
					"threads": 1,
					"rawintensity": 1024,
					"worksize": 16
				},
        {
					"index": 1,
					"corefreq": 1098,
					"memfreq": 500,
					"fanspeed": 60,
					"powertune": 20,
					"threads": 1,
					"rawintensity": 1024,
					"worksize": 16
				},
        {
					"index": 2,
					"corefreq": 1098,
					"memfreq": 500,
					"fanspeed": 60,
					"powertune": 20,
					"threads": 1,
					"rawintensity": 1024,
					"worksize": 16
				}
			],
			"pools":
			[
				{
					"url": "stratum+tcp://xmr-eu.dwarfpool.com:8100",
					"user": "XMRadress.TransactionID",
					"pass": "x"
				}
			]
		}
	]
}

**Comment by ZeroASIC** [comment]
Does it work on each GPU individually?

**Comment by kisk9** [comment]
Getting this same error with a single HD 5770 on Linux Mint 17.3 (Ubuntu 14.04). Tried with AMD SDK 2.9 and 3.0. CPU works fine as I imagine clGetDeviceIDs isn't even called if index = -1. Would love to get this thing started.

**Comment by AX-Turbo** [comment]
@ZeroASIC no, I can´t get any single card to work with this miner (0.3b).
Always the same error -1. With Claymore's XMR GPU miner my rig works immediately without problems.

**Comment by ZeroASIC** [comment]
I have the newest AMD driver and it's running great. I did compile myself so I do have the SDK.

**Comment by epigramx** [comment]
Has anyone figured it yet?

**Comment by technedigitus** [comment]
Same issue here, using Radeon 6670, Win7 64bits (last driver, opencld and SDK installed):

C:\wolf-xmr-miner-v0.4>miner.exe xmr.conf
[00:25:10] Setting up GPU(s).
[00:25:10] Error -1 when calling clGetDeviceIDs for number of devices.

C:\wolf-xmr-miner-v0.4>
